cookbook/external-media-storage.rst

```diff
@@ -0,0 +1,44 @@
+Store Media in an external Storage
+==================================
+
+Sulu is able to upload newly created media files directly to external storage provider (such as google cloud-storage).
+
+Currently available storages:
```

Comment:
I would say either the list contains link, or just omit it (the table of content gives an idea what is possible anyway)